Lib/test/test_sys.py

```diff
@@ -1046,7 +1046,7 @@ def test_getallocatedblocks(self):
         self.assertLessEqual(b, a)
         gc.collect()
         c = sys.getallocatedblocks()
-        self.assertIn(c, range(b - 50, b + 50))
+        self.assertLessEqual(c, b)
```

Comment:
There is the change for free-threaded build.
But IIUC, `c` is always less equal than `b`, but not sure why the range cooperation was applied that was not used when comparing `a` and `b`.